reposerver/repository/testdata/managed-by/managed-by.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: app
+  labels:
+    app.kubernetes.io/managed-by: another-maintainer
```

Comment:
`app.kubernetes.io/managed-by` does not indicate the maintainer. It indicates the tool used to manage the deployed resource.

> `app.kubernetes.io/managed-by` | The tool being used to manage the operation of an application.

https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/